scripts/benchmark_all.sh

```diff
@@ -156,7 +156,8 @@ do
     case "${config}" in
       "st") echo -n "single-threaded" ;;
```

Comment:
Should we include SF 10 in all title?

Reply:
Well, it's the title for all benchmark single-threaded runs, isn't it. We could add a branch for TPC-H, but I'd say that's not necessary. Now with the default of SF 10, that should be rather clear. At least I think it is.

Reply:
We don't communicate that SF 10 is the default though, do we?

Reply:
No. I would add SF10 to the titles as well.